src/Messages.js

```diff
@@ -692,6 +692,11 @@ export default defineMessages({
       'Used in the system table title column, the last time a system has checked in',
     defaultMessage: 'Last seen',
   },
+  noTags: {
+    id: 'nohits.noTags',
```

Comment:
Can we please keep translation key (`noTags`) and id (`nohits.noTags`) the same? Without the dot of course.
`noTags` -> `nohitsnoTags`